content/applications/websites/ecommerce/ecommerce_management/order_handling.rst

```diff
@@ -0,0 +1,112 @@
+
+Orders can be found under :menuselection:`Website --> eCommerce --> Orders/Unpaid Orders`. Each
+order goes through a different status:
+
+- **Quotation**: a new product has been added to the cart. This status represents all carts on the
+  website for which the customer has *not* gone through the checkout process yet;
+- **Quotation sent**: the customer has gone through the checkout process and confirmed the order but
+  the payment has not been confirmed yet;
+- **Order**: the customer has gone through the checkout process, confirmed the order, and the
+  payment has been received.
+
+.. image:: order_handling/handling-status.png
+   :align: center
+   :alt: Statuses of eCommerce orders
+
+Abandoned cart
+--------------
+
+An **abandoned cart** represents an order for which the customer did **not finish** the checkout
+confirmation process. For these orders, it is possible to automatically send an **email reminder**
+to the customer. To enable that feature, go to :menuselection:`Website --> Configuration -->
+Settings` and in the **Email & Marketing** section, enable :guilabel:`Automatically send abandoned
+checkout emails`. Once enabled, the **time-lapse** after which an email is automatically sent can be
+configured, as well as the **email template** used.
+
+.. note::
+   For an email to be sent, Odoo needs the contact information. For that, the customer must either
+   have entered their contact details during the checkout process; or be logged-in when they added
+   the product to their cart.
+
+.. _handling/delivery:
+
+Delivery orders
+===============
+
+Delivery flow
+-------------
+
+Once a quotation has been confirmed, a delivery order is automatically created. The next step is to
+process this delivery.
+
+Packing eCommerce orders usually requires to pick the product, prepare the packaging, print the
+shipping label(s) and ship to the customer. Depending on the number of orders, strategy, or
+resources, those steps can be considered as one or multiple actions in Odoo.
+
+An automatic email can be sent to the customer when the transfer in Odoo is at the “done” stage. The
+feature can be enabled in the settings of the
+:doc:`Inventory </applications/inventory_and_mrp/inventory>` app.
+
+.. note::
+   If customers are allowed to pay when picking up their order in stores or by wire transfer, the
+   quotation will **not** be confirmed and the stock will **not** be reserved. Orders must be
+   confirmed manually to reserve products in stock.
+
+.. seealso::
+   - :doc:`/applications/inventory_and_mrp/inventory/shipping/operation/invoicing`
+   - :doc:`/applications/inventory_and_mrp/inventory/shipping/operation/labels`
+   - :doc:`/applications/inventory_and_mrp/inventory/shipping/operation/multipack`
+
+Returns and refunds
+-------------------
+
+The only way for a customer to return an order is through an online form. Depending on the return
+strategy or type of product, it may not be possible to return products.
```

Comment:
```suggestion
A customer can only return an order through an online form. It may not be possible to return products depending on the return strategy or type of product.
```